src/scss/layout/_header.scss

```diff
@@ -2,26 +2,47 @@
 
 .site-header {
   // header section
+  position: relative;
   display: flex;
   justify-content: space-between;
-
+  height: rem(80px);
+  
+  &::after {
+    content: '';
+    position: absolute;
+    bottom: 0;
+    left: calc((100vw - 100%) / -2 - 10px);
+    display: block;
+    width: 100vw;
+    height: 1px;
+    background-color: $color__gray-700;
+  }
+  
   // logo link
   &__logo {
     display: flex;
     align-items: center;
+    margin: 0 auto;
     color: $color__white;
     font-weight: bold;
     line-height: 1.05;
```

Comment:
Can you add `@include font-size(16px);` here? We should explicitly set the font size in case the body size changes.